egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -922,7 +922,8 @@ def __init__(self,
                                  """, default="queue.pl")
         self.parser.add_argument("--egs.cmd", type=str, dest="egs_command",
                                  action=common_lib.NullstrToNoneAction,
-                                 help="Script to launch egs jobs")
+                                 help="Script to launch egs jobs",
+                                 default="queue.pl")
```

Comment:
This change is not right, it will break the example scripts.  
Look for statements like this:
```
    run_opts.egs_command = (args.egs_command
                            if args.egs_command is not None else
                            args.command)
```

Reply:
BTW I don't like the way the code is currently (as pointedout in my comment above),
 as in-place modification of variables is confusing.  I'd prefer it if where `args.egs_command` is used, we instead use
`(args.egs_command if args.egs_command is not None else args.command)`